src/hotspot/share/runtime/arguments.cpp

```diff
@@ -34,6 +34,7 @@
 #include "compiler/compilerDefinitions.hpp"
 #include "gc/shared/gcArguments.hpp"
 #include "gc/shared/gcConfig.hpp"
+#include "gc/shared/genArguments.hpp"
```

Comment:
Why is this needed?

Reply:
`Arguments::set_heap_size` accesses `OldSize`, which is declared in this header.

Reply:
Got it. Thanks